src/sasctl/pzmm/write_json_files.py

```diff
@@ -499,7 +481,7 @@ def calculateFitStat(
             working directory.
 
         Yields
-        ---------------
+        ------
```

Comment:
Same issue as with writeVarJSON() docstring, the Yields section is inappropriate here.

Reply:
Resolved.